docs/Modules/multichain.mdx

```diff
@@ -15,44 +18,43 @@ import TabItem from '@theme/TabItem';
 <TabItem value="js" label="npm">
 
 ```bash
-
 npm install @biconomy/modules 
-
 ```
 
 </TabItem>
 <TabItem value="py" label="yarn">
 
 ```bash
-
 yarn add @biconomy/modules 
-
 ```
 
 </TabItem>
 <TabItem value="java" label="pnpm">
 
 ```bash
-
 pnpm add @biconomy/modules 
-
 ```
 
 </TabItem>
 </Tabs>
 
 ## Integration 
+In this, we describe the flow for integrating Multichain validation module. Let's start by looking at the code:
 
 ```typescript
-
 import { MultiChainValidationModule, DEFAULT_MULTICHAIN_MODULE } from "@biconomy/modules";
 
-    const multiChainModule = await MultiChainValidationModule.create({
-    signer: signer,
-    moduleAddress: DEFAULT_MULTICHAIN_MODULE
-  })
+const multiChainModule = await MultiChainValidationModule.create({
+  signer: signer,
+  moduleAddress: DEFAULT_MULTICHAIN_MODULE
+})
 ```
-Once initialized it can be passed to the smart account [create method](/Account/methods/create) to create a smart account. 
+As per the code, 
+- First we import the module and its associated address.
+- Utilize the create method to initialize the module. This method requires a signer, and the module address as required arguments.
+Biconomy supports various [third party signer](/Account/signers/description) integrations so you can use any signer of your choice like viem, privy etc.
+- Once initialized it can be passed to the smart account [create method](/Account/methods/create) to create a smart account. 
+This module can be used both as default and active validation module.
```

Comment:
```suggestion
As per the code, 
- First we import the module and its associated address.
- Utilize the create method to initialize the module. This method requires a signer, and the module address as required arguments.
Biconomy supports various [third party signer](/Account/signers/description) integrations so you can use any signer of your choice like viem, privy, etc.
- Once initialized it can be passed to the smart account [create method](/Account/methods/create) to create a smart account. 
This module can be used both as a default and active validation module.
```